node/clients/checkout.ts

```diff
@@ -191,15 +192,15 @@ export class Checkout extends JanusClient {
       .catch(statusToError) as Promise<IOResponse<OrderForm>>
   }
 
-  public changeToAnonymousUser = () => {
-    const { orderFormId } = this.context as CustomIOContext
-
-    if (!orderFormId) {
-      throw new Error('Missing orderFormId. Use withOrderFormId directive.')
-    }
-
+  public changeToAnonymousUser = (orderFormId: string) => {
     return this.get(this.routes.changeToAnonymousUser(orderFormId), {
       metric: 'checkout-change-to-anonymous',
+    }).catch((err) => {
+      // This endpoint is expected to return a redirect to
+      // the user, so we can ignore the error if it is a 3xx
+      if (!err.response || /^3..$/.test((err as AxiosError).code ?? '')) {
```

Comment:
I suggest you to improve readability to set the error type on the function parameter (line 198)
```ts
// line 198
.catch((err: AxiosError) => {
```
```suggestion
      if (!err.response || /^3..$/.test(err.code ?? '')) {
```

Reply:
@igorbrasileiro I think he may have done a bad rebase, you can see there are some of my commits in this branch which already are in `master`